### PR TITLE
Add some more things to member.js

### PR DESCRIPTION
**Pull request by LunarTwilight**
update member.js for some things

### CODE REVIEW

**Comment by Sidemen19** [comment]
You should probably link to Special:VerifyUser on CC as dev doesn't have the script imported. Tis what Wiki-Bot does for its verification system too.

**Comment by LunarTwilight** [comment]
Dev might end up importing the script, but if it doesn't, I'll update the link to use CC instead.

File: src/plugins/commander/commands/member.js [hunk 1]
```diff
@@ -26,7 +26,11 @@ class MemberCommand extends Command {
     }
 
     getMastheadDiscord(userid) {
-        return got(`https://services.fandom.com/user-attribute/user/${userid}/attr/discordHandle`).json();
+        return got(`https://services.fandom.com/user-attribute/user/${userid}/attr/discordHandle`, {
+            headers: {
+                accept: '*/*'
```
Comment (puxlit) [inline src/plugins/commander/commands/member.js:31]: This was discussed [here](https://discordapp.com/channels/246075715714416641/476452336282107925/761916568782241802), but to summarise, it appears the response content type is `application/hal+json`. When using Got's `.json()` method, the `Accept` request header is implicitly set to `application/json`, which results in a 406 (on cache miss). Thus, setting the `Accept` request header explicitly to either `application/hal+json` or `*/*` remediates this issue.

File: src/plugins/commander/commands/member.js [hunk 2]
```diff
@@ -51,10 +55,11 @@ class MemberCommand extends Command {
         const verifyUser = await this.getMastheadDiscord(editsAndID.query.users[0].userid);
 
         if (verifyUser.value !== message.author.tag) {
-            return message.channel.send('The username and tag in the masthead does not match the username and tag of the message author.');
+            return message.channel.send(`The username and tag in the masthead does not match the username and tag of the message author. Use <https://dev.fandom.com/wiki/Special:VerifyUser/${content}?user=${message.author.username}&tag=${message.author.discriminator}&c=!member&ch=lobby>.`);
```
Comment (puxlit) [inline src/plugins/commander/commands/member.js:58]:
```suggestion
            return message.channel.send(`The username and tag in the masthead does not match the username and tag of the message author. Use <https://dev.fandom.com/wiki/Special:VerifyUser/${encodeURIComponent(content)}?user=${encodeURIComponent(message.author.username)}&tag=${message.author.discriminator}&c=!member&ch=lobby>.`);
```

**Review by puxlit — CHANGES_REQUESTED** [review]

**Review by puxlit — APPROVED** [review]
👍

**Review by Dorumin — APPROVED** [review]
yep it's cool all good you're a weeb merging

**Pull request merged**